Add promfana helm docs (#9858)
hashicorp/consul: website/content/docs/k8s/observability/metrics.mdx

## Metrics in the UI Topology Visualization

## Deploying Prometheus and Grafana (_for demo and non-production use-cases only_)

> The helm chart now contains manifests for Prometheus and Grafana. They can be selectively installed during the Helm install depending on the values of `prometheus.enabled` and `grafana.enabled`. These manifests are based on the community manifests for Prometheus and Grafana.

Reviewer: ```suggestion
The Helm chart now contains manifests for Prometheus and Grafana. They can be installed with Helm via `prometheus.enabled` and `grafana.enabled`. These manifests are based on the community manifests for Prometheus and Grafana.
```
